curriculum/js2/3.js

```diff
@@ -5,8 +5,18 @@
  * @returns {array}
  */
 
-const solution = (a) => {
-  return [];
+const isPrime = require('../js1/7.js').solution;
+
+const solution = (a, idx = 0) => {
+  if (idx === a.length) {
```

Comment:
if your `if` condition only has one line, you don't need `{}`. so your code could be as simple as:
```
const solution = (a, idx = 0) => {
  if(idx === a.length) return a;
  if(!isPrime(a[idx])) a[idx] = 1;
  return solution (a, idx + 1);
}
```